### DISCUSSION TITLE
How to delete logs prior to specific date

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#9685] [Unsupported Versions]
I have a ManagerSearch server that the disk keeps filling up.  I've reviewed the Curator documentation and made edits to the global.sls 'close' and 'delete' settings, but it seems to have no affect.  Now I'm trying to run soup and it's pulling lots of containers, but there's not enough disk space.  Is there a command that I can manually run to delete logs prior to a certain date and keep let's say the last 30 days only?  I've mistakenly ran 'so-elastic-clear' before and it deletes ALL the logs, which is not ideal.  Please advise.

**Comment by cm-ops** [comment]
What is your log_size_limit set to in the managersearch pillar? `salt-call pillar.get elasticsearch:log_size_limit` will give you that. You could try lowering that to clean up Elasticsearch logs sooner. Default should be 80% of `/nsm`.

**Reply by innovate-support** [reply]
Thanks, I'll give that setting a try.

**Reply by innovate-support** [reply]
That setting does in fact work as expected and I can now manually set the size limit to my liking.  Thanks so much!
